tools/mzml_validator/.shed.yml

```diff
@@ -0,0 +1,14 @@
+owner: recetox
+remote_repository_url: "https://github.com/RECETOX/galaxytools/tree/master/tools/mzml_validator"
+homepage_url: "https://github.com/RECETOX/galaxytools"
+categories:
+  - Metabolomics
+  - Proteomics
+repositories:
+  mzml_validator:
+    description: "mzML Validator checks if mzML file validates against XML Schema Definition of HUPO Proteomics Standard Initiative."
+    include:
```

Comment:
I would remove this, its an additional step to keep up to date and is easy to miss.

Reply:
What do you mean specifically? The description or the include?

Reply:
Sorry, the entire include section. We do not use that a lot, it's more the exception to use it than the default.

Reply:
Is everything from the folder automatically included, so is that part optional? How does it work if this part is not in the shed.yaml?

Reply:
Everything is included by default. If you have files you don't want to ship you can exclude them. As usual have a look at the IUC repo for inspiration: https://github.com/galaxyproject/tools-iuc

Reply:
Didn't know that - thanks for the hint!